src/main/java/com/labsynch/labseer/chemclasses/rdkit/BBChemStructureServiceImpl.java

```diff
@@ -80,8 +80,7 @@ public void populateDescriptors(RDKitStructure rdKitStructure) {
 		// canonical	: if false, no attempt will be made to canonicalize the SMILES
 		// allBondsExplicit	: if true, symbols will be included for all bonds.
 		// allHsExplicit	: if true, hydrogen counts will be provided for every atom.
-		// doRandom : undocumented as of 2021-11-29 but default is false
-		rdKitStructure.setSmiles(RDKFuncs.MolToSmiles(mol, true, false, -1, true, false, false, false, false));
+		rdKitStructure.setSmiles(RDKFuncs.MolToSmiles(mol, true, false, -1, true, false, false, false));
```

Comment:
Ok so now you fixed this line but also removed the comment

Reply:
Sorry, went back and forth on this a few times.  I should have checked the code for compilation before adding the additional doRandom variable.  This variable doesn't exist in the java wrappers and won't compile with it.  I'm leaving it out of this pull request.

Reply:
I think if you count the number of arguments, you are already providing `doRandom` on line 83, you just don't have it in your docstring